python/pyspark/sql/functions.py

```diff
@@ -206,17 +227,20 @@ def _():
 @since(1.3)
 def approxCountDistinct(col, rsd=None):
     """
-    .. note:: Deprecated in 2.1, use approx_count_distinct instead.
+    .. note:: Deprecated in 2.1, use :func:`approx_count_distinct` instead.
     """
     return approx_count_distinct(col, rsd)
 
 
 @since(2.1)
 def approx_count_distinct(col, rsd=None):
-    """Returns a new :class:`Column` for approximate distinct count of ``col``.
+    """Returns a new :class:`Column` for approximate distinct count of column `col`.
 
-    >>> df.agg(approx_count_distinct(df.age).alias('c')).collect()
-    [Row(c=2)]
+    :param rsd: Residual (float). The approximate count will be within this fraction of the true
```

Comment:
For this one, let's add the contents in Scala/R one too - https://github.com/apache/spark/blob/3a840048ed3501e06260b7c5df18cc0bbdb1505c/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L245 and https://github.com/apache/spark/blob/7f203a248f94df6183a4bc4642a3d873171fef29/R/pkg/R/functions.R#L2243

This one in Scala/R look already having the param contents to match with.